1/test/s2i/configuration/config.xml

```diff
@@ -48,9 +47,9 @@
       <properties class="hudson.model.View$PropertyList"/>
     </hudson.model.AllView>
   </views>
-  <primaryView>All</primaryView>
+  <primaryView>s2i-test-config</primaryView>
   <slaveAgentPort>49187</slaveAgentPort>
-  <label>master</label>
+  <label>s2i-test-config</label>
```

Comment:
Pretty positive it isn't the sync plugin.  It can save jenkins config for the new pod template stuff, but we are not manipulating pod templates in the s2i test, and even if it was, and the actual reading / writing of the config is done by core jenkins API.  

More likely a change in core jenkins, or one of its dependencies, when moving to v2.60.1, where their XML processing, when consuming the config.xml, is stripping the comment.

Still, good to know.  Thanks for the heads up.